### ISSUE TITLE
Roidmi 4.22

### ISSUE THREAD

**Issue by kkszysiu** [kkszysiu/mifm#12]
Hey guys,

I prepared experimental support for new version of RoidMi.

Sad part is that they changed almost whole UI and all old translations are incompatible.

Good thing is that I found some translation on English in new sources, but it's pretty bad translation and not polished at all. I prepared new tags for new version starting from 1.x.x.

I think during time we will polish translations.

I'm currently working on adding Polish translation.

@vivalaflam do you think you can improve English translation?
@IgorTimoshenko can you help with Ukrainian translation?

Thanks!

**Comment by lszeremeta** [comment]
:+1:

**Comment by vivalaflam** [comment]
Yes I can totally do it now that we have the Orthodox Easter vacation

**Comment by ihortymoshenko** [comment]
@kkszysiu, yes, I can :)
